delivery_carrier_label_paazl/tests/test_delivery_carrier_label_paazl.py

```diff
@@ -114,6 +114,11 @@ def _request(self, path, method="POST", data=None, headers=None):
 class TestDeliveryCarrierLabelPaazl(
         DeliveryCarrierLabelPaazlCase,
         carrier_label_case.TestCarrierLabel):
+    def _picking_data(self):
+        """Add the boxes that vanmoof_shipping module makes required for Paazl"""
+        self.picking.paazl_pack_detail_ids = [(0, 0, {"name": __name__})]
```

Comment:
Here too, remove Vanmoof specifics. Can you explain why you think this is necessary?

Reply:
I think Martin already discussed this with you, but the idea is that we expect the tests of this module to fail on a database that has vanmoof_shipping installed, indeed because of the additional restrictions. We only need to be these tests green in isolation.

Reply:
Got it, clear then, will remove them and reproduce case locally w/o vanmoof_shipping tests affection then.